src/lib/AccountStoreIndexeddb.js

```diff
@@ -69,14 +68,15 @@ class AccountStore {
                 if (cursor) {
                     const key = cursor.value;
 
-                    // Because: To use Key.getPublicInfo(), we would need to create Key instances out of the key object that we receive from the DB.
-                    const keyInfo = {
+                    // Because: To use Key.getPublicInfo(), we would need to create Key
+                    // instances out of the key object that we receive from the DB.
+                    const accountInfo = {
```

Comment:
The account store is in the keyguard for legacy reasons only, right?

Reply:
Yes. Will be removed after the transition period.